_config.yml

```diff
@@ -54,15 +54,15 @@ languages: ["en", "de", "pl", "ru"]
 exclude_from_localizations: ["assets", "admin", "vendor"]
 header_themes:
   - url: /about
-    text: about
+    key: about.label
   - url: /news
-    text: articles
+    key: news.label
   - url: /press
-    text: Press
+    key: press
   - url: /events
-    text: events
+    key: events.label
   - url: /works-councils
-    text: Works Councils
+    key: works_councils.label
```

Comment:
This broke the text of menu, needs to be removed or have code updated first

Reply:
I did not mean to push changes to this file, my bad